docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
If a delegated OCSP signing certificate is compromised you can use that to make any revoked or unknown certificate valid. So basically no certificate issued by the CA can be trusted as your are not sure if you can trust the revocation status and there is no good way to revoke the signing certificate except from letting it expire. And no signing certificates should not have CA:TRUE.

30 day sounds fair, probably you can limit the key usage to OCSP signing only?

Yes I would like the requirement to pre-produce to be removed.

I would not require the extent the revocation status to the beginning, non TLS services could still trust the certificate until revocation when time stamped. Also this would require back-dating of the revocation. 

The minimum validity is 8 hours, we reproduce every hour with a longer validity but only for certificates that are queried.

With 3.5 days my cached request spikes could increase by 84x. Probably not impossible to handle on the CDN but we are talking about spikes of tens of thousands of requests per second where it's currently nicely distributed. In the case of a CDN failure more users might be impacted as requests are all concentrated in a much shorter period.